scripts/consumer_apps_builder.py

```diff
@@ -100,7 +106,11 @@ def assemble_apk(domain, build_number, username, password, build_type):
 
 def get_app_name_from_profile():
     tree = ET.parse(PATH_TO_ASSETS_DIR_FROM_ODK + '/direct_install/profile.ccpr')
-    return tree.getroot().get("name")
+    return escape_apostrophes(tree.getroot().get("name").encode('utf-8'))
+
+
+def escape_apostrophes(s):
+    return s.replace("'", "\\'")
```

Comment:
No, quotes would have the same issue, as would a bunch of other punctuation. I decided not to go overboard with adding every possible case here, since apostrophes are the only one that showed up in an application name so far (and seem like the only one likely to).

Adding the comment to the file, though